mslice/cli/helperfunctions.py

```diff
@@ -154,6 +154,8 @@ def is_slice(*args):
     Checks if args[0] is a WorkspaceBase or HistogramWorkspace
     """
     if len(args) > 0:
```

Comment:
Can we have this function `return` either True or False in all cases rather than returning `None` or raising an exception? `is_`-type functions should return True/False as that's what a caller would expect.

Reply:
I think something similar for `is_cut` would be good.